manager/src/main/resources/application.properties

```diff
@@ -2,9 +2,9 @@ event-bridge.k8s.orchestrator=${EVENT_BRIDGE_K8S_ORCHESTRATOR:openshift}
 event-bridge.manager.url=${EVENT_BRIDGE_MANAGER_URL:http://localhost:8080}
 
 # SLI Metric Names
-rhose.metrics-name.operation-total-count=managed_services_api_rhose_operation_count
-rhose.metrics-name.operation-success-total-count=managed_services_api_rhose_operation_success_count
-rhose.metrics-name.operation-failure-total-count=managed_services_api_rhose_operation_failure_count
+rhose.metrics-name.operation-total-count=managed_services_api_rhose_operation_total_count
```

Comment:
@manstis Can I just double check that this doesn't result in the metric being exposed as `managed_services_api_rhose_operation_total_count_total` for example? I seem to remember that when I did this originally, the `_total` suffix was automatically appended by Micrometer framework, hence why it's missing from the metric name. 

For example if I check `dev` environment now, I see the following metrics exposed as an example:

```
managed_services_api_rhose_operation_success_count_total{operation="provision",resource="processor",} 146.0
managed_services_api_rhose_operation_success_count_total{operation="delete",resource="processor",} 146.0
managed_services_api_rhose_operation_success_count_total{operation="delete",resource="bridge",} 218.0
managed_services_api_rhose_operation_success_count_total{operation="provision",resource="bridge",} 220.0
managed_services_api_rhose_operation_success_count_total{operation="modify",resource="processor",} 30.0
managed_services_api_rhose_operation_count_total{operation="provision",resource="processor",} 146.0
managed_services_api_rhose_operation_count_total{operation="delete",resource="processor",} 146.0
managed_services_api_rhose_operation_count_total{operation="delete",resource="bridge",} 218.0
managed_services_api_rhose_operation_count_total{operation="modify",resource="bridge",} 13.0
managed_services_api_rhose_operation_count_total{operation="provision",resource="bridge",} 220.0
managed_services_api_rhose_operation_count_total{operation="modify",resource="processor",} 30.0
```

cheers,
Rob

Reply:
... and now it looks like the code was already correct and the spreadsheet was out of date...

Closing this PR (as there are no changes!).

Reply:
Thanks @manstis 👍